plugins/modules/cloud/xenserver/xenserver_guest.py

```diff
@@ -63,7 +63,6 @@
     - In case of multiple VMs with same name, use C(uuid) to uniquely specify VM to manage.
```

Comment:
I believe that by new guidelines this line:

`from __future__ import absolute_import, division, print_function`

should be:

`from __future__ import (absolute_import, division, print_function)`

Reply:
@bvitnik which guidelines do you mean?

Reply:
All the examples in Ansible Developer Guide use this form. I can also vaguely recall that there was a massive commit to change this line in some collection of modules. It prompted me to also change the line. I'm really not sure any more :).

Reply:
Ah ok. The sanity tests accept both versions, so I guess both are ok. I just looked at ansible/ansible, there every file in lib/ansible uses `from __future__ import (absolute_import, division, print_function)`, **except** all modules, some module_utils, plugins/filter/urls.py and plugins/doc_fragments/vars_plugin_staging.py: these use `from __future__ import absolute_import, division, print_function`. So... no idea :)